From 5f8259fb66d4a2e15febce9a1713bacde31e6399 Mon Sep 17 00:00:00 2001
From: jenisys <jenisys@users.noreply.github.com>
Date: Sun, 9 Jun 2019 13:17:10 +0200
Subject: [PATCH] py3.8: Try to fix logging.Formatter validate problem

---
 tests/unit/test_capture.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/unit/test_capture.py b/tests/unit/test_capture.py
index ac2655e..d9a3f3a 100644
--- a/tests/unit/test_capture.py
+++ b/tests/unit/test_capture.py
@@ -20,6 +20,8 @@ def create_capture_controller(config=None):
         config.log_capture = True
         config.logging_filter = None
         config.logging_level = "INFO"
+        config.logging_format = "%(levelname)s:%(name)s:%(message)s"
+        config.logging_datefmt = None
     return CaptureController(config)
 
 def setup_capture_controller(capture_controller, context=None):
