From e55e2daf6def5a80a34469640e3ef3c1938c5ea8 Mon Sep 17 00:00:00 2001
From: jenisys <jenisys@users.noreply.github.com>
Date: Sun, 9 Jun 2019 19:24:14 +0200
Subject: [PATCH] Comment tweaks

---
 behave/runner.py | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/behave/runner.py b/behave/runner.py
index f0f077a..f209cb0 100644
--- a/behave/runner.py
+++ b/behave/runner.py
@@ -167,10 +167,15 @@ class Context(object):
         self._record = {}
         self._origin = {}
         self._mode = self.BEHAVE
+
+        # -- MODEL ENTITY REFERENCES/SUPPORT:
         self.feature = None
-        # -- RECHECK: If needed
+        # DISABLED: self.rule = None
+        # DISABLED: self.scenario = None
         self.text = None
         self.table = None
+
+        # -- RUNTIME SUPPORT:
         self.stdout_capture = None
         self.stderr_capture = None
         self.log_capture = None
