From cc283f40c6fcefa992ddb54b60adb79330abae34 Mon Sep 17 00:00:00 2001
From: jenisys <jenisys@users.noreply.github.com>
Date: Wed, 11 Dec 2019 08:37:11 +0100
Subject: [PATCH] Tweak pytest configuration to silence JUnit XML dialect
 warning.

---
 pytest.ini | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/pytest.ini b/pytest.ini
index ff2a8a2..228279c 100644
--- a/pytest.ini
+++ b/pytest.ini
@@ -16,9 +16,10 @@
 # ============================================================================
 
 [pytest]
-minversion    = 2.8
+minversion    = 4.2
 testpaths     = tests
 python_files  = test_*.py
+junit_family = xunit2
 addopts = --metadata PACKAGE_UNDER_TEST behave
     --metadata PACKAGE_VERSION 1.2.7.dev1
     --html=build/testing/report.html --self-contained-html
@@ -27,6 +28,10 @@ markers =
     smoke
     slow
 
+# -- PREPARED:
+# filterwarnings =
+#    ignore:.*invalid escape sequence.*:DeprecationWarning
+
 # -- BACKWARD COMPATIBILITY: pytest < 2.8
 # norecursedirs = .git .tox build dist py.requirements tmp* _WORKSPACE
 
