From a4264920f7852cec33d1d1679310c194a23e8c67 Mon Sep 17 00:00:00 2001
From: jenisys <jenisys@users.noreply.github.com>
Date: Mon, 13 Apr 2020 10:31:54 +0200
Subject: [PATCH] FIX: Cross-reference problem (copy+paste) in Rule class
 docstring.

---
 behave/model.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/behave/model.py b/behave/model.py
index f46d2c2..cb69f9e 100644
--- a/behave/model.py
+++ b/behave/model.py
@@ -84,8 +84,8 @@ class ScenarioContainer(TagAndStatusStatement, Replayable):
 
     .. attribute:: keyword
 
-       This is the keyword as seen in the *feature file*. In English this will
-       be "Feature" or "Rule".
+       This is the keyword as seen in the *feature file*.
+       In English this will be "Feature" or "Rule".
 
     .. attribute:: name
 
@@ -671,7 +671,7 @@ class Rule(ScenarioContainer):
 
 
     .. versionadded:: 1.2.7
-    .. _`feature`: gherkin.html#rule
+    .. _`rule`: gherkin.html#rule
     """
     type = "rule"
 
