From 8396beb25a4dc6a9da0cc01f285a0d03cc0b6856 Mon Sep 17 00:00:00 2001
From: Renovate Bot <bot@renovateapp.com>
Date: Wed, 4 Nov 2020 08:43:32 +0000
Subject: [PATCH] Add renovate.json

---
 renovate.json | 5 +++++
 1 file changed, 5 insertions(+)
 create mode 100644 renovate.json

diff --git a/renovate.json b/renovate.json
new file mode 100644
index 0000000..f45d8f1
--- /dev/null
+++ b/renovate.json
@@ -0,0 +1,5 @@
+{
+  "extends": [
+    "config:base"
+  ]
+}
